### ISSUE TITLE
Revert settings in VS Code

### ISSUE THREAD

**Issue by isolin** [JosefPihrt/Roslynator#719]
I installed Roslynator for VS Code today. It listed around 9K suggestions for my project. Some of them were really useful, but the editor started to lag trying to handle so many issues so that it was impossible to type anything. My computer is a few years old, yet this is not a reason for me to buy a new one. So I uninstalled Roslynator from VS Code and rebooted.

The issue is that the editor keeps showing around 2K of `csharp(IDExxxx)` infos with style suggestions which still affect the performance so that normal coding is not feasible. That was not the case before I installed Roslynator. I checked the VS Code Settings: native Roslyn Code Analysis is `off` now and I set OmniSharp to skip `info` level messages. It did not help.

So my question is: What was it that Roslynator changed and did not revert upon uninstall? I wish to be able to use VS Code fluently again, please help.

**Comment by isolin** [comment]
Finally, I was able to resolve the issue. I found out that the file `~/.omnisharp/omnisharp.json` contained a directive that enabled  Roslyn Analyzers. This was certainly not the case before I installed Roslynator. It is also quite a nasty way to enable them, since this setting is not exposed to the VS Code interface and it silently overrides the user settings.

**Comment by JosefPihrt** [comment]
`omnisharp.json` is updated during installation of Roslynator. It is only way make Roslynator actually working. If the `omnisharp.json` would not be updated people would be complaning that Roslynator is not working and that they have to update `omnisharp.json` manually.

And the configuration in `omnisharp.json` should be cleaned up on uninstall (see #666).

**Comment by AdrianWilczynski** [comment]
Hi.

Uninstall script only removes Roslynator entries from `LocationPaths`. It doesn't revert `EnableAnalyzersSupport`. I figured that somebody might be using some other analyzers, so it's kinda a hard call to make whether it should get set to false or not.

"_it silently overrides the user settings_"

It's not really silent. There is a pop-up saying `omnisharp.json has been updated with Roslynator configuration.`
